app/presenters/publishing_api/payload_builder/people.rb

```diff
@@ -0,0 +1,44 @@
+module PublishingApi
+  module PayloadBuilder
+    class People
+      attr_reader :item, :key
+
+      def self.for(item, key)
+        self.new(item, key).call
+      end
+
+      def initialize(item, key)
+        @item = item
+        @key = key
+      end
+
+      def call
+        people
+      end
+
+    private
+
+      def people
+        return {} unless people_present?
+
+        { key => people_from_role_appointments + person_from_role_appointment }
+      end
+
+      def people_present?
+        people_from_role_appointments.present? || person_from_role_appointment.present?
+      end
+
+      def people_from_role_appointments
```

Comment:
👍  good idea

My take on that ends up like this:
2a6a5e3766e992f60b45aa147b5a048e431cde20
5ea077e49067e40d71d2f55c7565850c5fc74441

I think it cleans the code up a bit.
I've kept the idea of `people` and `role_appointments`, for example in two separate methods because I think it expresses what the code is doing fairly clearly, but could squash it into one?